src/pages/storage/CustomVolumeModal.tsx

```diff
@@ -34,14 +37,25 @@ const CustomVolumeModal: FC<Props> = ({
     setPrimaryVolume(volume);
   };
 
+  const handleGoBack = () => {
+    setContent(SELECT_VOLUME);
+  };
+
+  let modalTitle = title ?? "Choose custom volume";
+  if (content === CREATE_VOLUME) {
+    modalTitle = title ? (
+      <BackLink
+        title={"Create volume"}
+        onClick={handleGoBack}
+        linkText={"Attach custom volume device"}
```

Comment:
```suggestion
        linkText="Attach custom volume"
```
This can be a bit shorter, and can you remove the `{` from all the string values?